extensions/amp-story/1.0/page-advancement.js

```diff
@@ -1039,6 +1042,9 @@ export class MediaBasedAdvancement extends AdvancementConfig {
       this.video_ = video;
     });
 
+    // Removes [loop] attribute if specified, so the 'ended' event can trigger.
+    this.element_.querySelector('video').removeAttribute('loop');
```

Comment:
consider changing this to:
```
ampVideoEl.getImpl().then(ampVid => {
  ampVid.propagateAttributes(
            'loop',
            ampVideoEl.firstElementChild,
            /* opt_removeMissingAttrs */ true
          );
        });
```

since i've seen it used in the codebase for this use case

Reply:
Cool, thank you 👍

Reply:
I reverted this as `propagateAttributes` and `element` are not allowed by the video interface